src/Common/src/Interop/Windows/mincore/Interop.UnsafeCreateFile.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [System.Security.SecurityCritical]  // auto-generated
+        internal static SafeFileHandle UnsafeCreateFile(
```

Comment:
This exists just to add an "Unsafe" prefix to something that already exists?  Do we really need this, or can the rest of the code just use CreateFile and SafeCreateFile?